tests/test_utils.py

```diff
@@ -21,12 +21,31 @@ def test_partial_workflow(caplog, multiple_named_steps_workflow):
     for step_num in range(num_steps):
         steps_by_name = \
             get_targeted_workflow_steps(
-                workflow_spec, f'{step_prefix}{step_num}')
+                workflow_spec, f'{step_prefix}{step_num}', None)
         assert len(steps_by_name) == step_num + 1
     # User does not specify a target
-    assert len(get_targeted_workflow_steps(workflow_spec, None)) == \
+    assert len(get_targeted_workflow_steps(workflow_spec, None, None)) == \
```

Comment:
get_targeted_workflow_steps don't need `None` optional keyword arguments anymore, here and elsewhere in the PR